pymove/_version.py

```diff
@@ -463,8 +423,7 @@ def render_git_describe(pieces):
 
 
 def render_git_describe_long(pieces):
-    """
-    TAG-DISTANCE-gHEX[-dirty].
+    """TAG-DISTANCE-gHEX[-dirty].
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)